source-container-build/app/source_build.py

```diff
@@ -211,7 +215,12 @@ def skopeo_copy(
         flags.append("--remove-signatures")
     cmd = ["skopeo", "copy", *flags, src, dest]
     logger.debug("copy image: %r", cmd)
-    run(cmd, check=True)
+    try:
+        run(cmd, check=True, capture_output=True)
```

Comment:
By adding `capture_output=True`, python will eat the stdout and stderr of `skopeo copy`. IMO we should log them both on failure, otherwise debugging will be harder